### DISCUSSION TITLE
Adding file upload feature to Github Mobile App

### DISCUSSION THREAD

**Discussion by leecheeyong** [community/community#112768] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

Seems like it is possible to create a text file in Github Mobile app, I don't see why not for file upload, like media, video, .etc
I'm an iOS user, this feature isn't available on the app, and yes I know it is possible on any browser but it'd be great if this tiny feature could be implemented, thanks

**Comment by Bvazor** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Seems like it is possible to create a text file in Github Mobile app, I don't see why not for file upload, like media, video, .etc
> I'm an iOS user, this feature isn't available on the app, and yes I know it is possible on any browser but it'd be great if this tiny feature could be implemented, thanks

**Comment by esahmed21** [comment, marked as answer]
Thank you for your insightful suggestion, @leecheeyong! Your idea is truly valuable and will definitely be added to our backlog for further discussion with the team.

**Reply by leecheeyong** [reply]
It seems like it has been a little while now and this feature has not been added yet unfortunately

**Comment by leecheeyong** [comment]
> Thank you for your insightful suggestion, @leecheeyong! Your idea is truly valuable and will definitely be added to our backlog for further discussion with the team.
> 
> 

Thanks for the reply, looking forward to this feature being added soon

**Comment by MrNiSso** [comment]
🔴Can u ratings my app and give me to /10🌟

https://play.google.com/store/apps/details?id=com.fd.food_recipe0

**Comment by david1234121** [comment]
Z
